library/network/src/lib/y2firewall/firewalld/interface.rb

```diff
@@ -0,0 +1,81 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2018 SUSE LLC
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE.
+#
+# To contact SUSE about this file by physical or electronic mail, you may find
+# current contact information at www.suse.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "y2firewall/firewalld"
+
+module Y2Firewall
+  class Firewalld
+    # Class to work with firewalld interfaces
+    class Interface
+      # @return [Symbol]
+      attr_accessor :id
+
+      # Constructor
+      #
+      # @param name [String] interface name
+      def initialize(name)
+        Yast.import "NetworkInterfaces"
+
+        @id = name.to_sym
+      end
+
+      # Return an array with all the known or configured interfaces
```

Comment:
*know or configured*? I would like to have some clarification about what is exactly a "known" interface.